test-kustomizations.sh

```diff
@@ -60,7 +60,14 @@ mapfile -t FILES < <(find "${DIRS_TO_CHECK[@]}" \
 for FILE in "${FILES[@]}"; do
     DIRECTORY=$(dirname "${FILE}")
 
-    if ! OUTPUT=$("${COMMAND}" build "${DIRECTORY}" 2>&1); then
+    if [ $(grep -ce "^${DIRECTORY}$" .ci/relax_loader.txt) -eq 1 ]; then
```

Comment:
When I run this locally, it need to change `[ ]` to `[[ ]]` or else I get:

```
../test-kustomizations.sh: line 63: [: -eq: unary operator expected
```

Reply:
Also, even after the fix, the output is somewhat verbose:

```
grep: .ci/relax_loader.txt: No such file or directory
./common/metallb/kustomization.yaml: OK
grep: .ci/relax_loader.txt: No such file or directory
./common/nmstate/kustomization.yaml: OK
grep: .ci/relax_loader.txt: No such file or directory
./common/olm/kustomization.yaml: OK
```

Maybe we could suppress the `grep` message, or do you think that is desirable to see/know?

Reply:
Which shell do you use?

Reply:
Please do not look at this file. I guess David copied it from Pragadeeswaran, but I already requested changes there and it has better shape now: https://github.com/openstack-k8s-operators/architecture/pull/83/files#diff-1660d5268f2b4dcabf03d84bd0caf8486d4e73ded11d38920618e1fdd2bc78b4

Reply:
Understood.  Could we remove this from the PR, then?

Reply:
TL;DR This change does not belong here; @dsariel please restore the file to the shape from current `main` branch

Reply:
changed

Reply:
I still see it in current commit @dsariel :wink: